meta-balena-common/recipes-core/systemd/timeinit/fake-hwclock.service

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Balena Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[Unit]
+Description=Restore/save the current clock
+Documentation=man:fake-hwclock(8)
+DefaultDependencies=no
+Requires=bind-etc-fake-hwclock.service
+Wants=time-set.target
```

Comment:
`Wants=time-set.target` seems to be in a logical conflict with `WantedBy=time-set.target` and `Before=time-set.target`. I would expect systemd to deal with it but anyway - is this really intended?

Reply:
I added the `Wants=time-set.target` because the systemd examples that I found regarding how to add a service to a target show both `Wants=xxx.target` and `Before=xxx.target`.

The systemd documentation says the following about the time-set.target: `Services responsible for setting the system clock from a local source (such as a maintained timestamp file or imprecise real-time clock) should pull in this target and order themselves before it.`

Removing the `WantedBy=time-set.target` entry results in the service not starting.

Reply:
@markcorbinuk  did you try with the `WantedBy` and not the `Wants`?

Reply:
@alexgg it looks like everything works without the `Wants`, so I'll drop them.